packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -265,6 +265,12 @@ interface DatabaseClusterBaseProps {
   * @default - if storageEncrypted is true then the default master key, no key otherwise
   */
   readonly storageEncryptionKey?: kms.IKey;
+  /**
+  * Whether to copy tags to the snapshot when a snapshot is created.
+  *
+  * @default: true
+  */
+  readonly copyTagsToSnapshot?: boolean;
```

Comment:
This looks misaligned:

```suggestion
  /**
   * Whether to copy tags to the snapshot when a snapshot is created.
   *
   * @default: true
   */
  readonly copyTagsToSnapshot?: boolean;

Reply:
Updated the comment as per suggestion.